fluentgen/src/main/java/com/azure/autorest/fluent/mapper/FluentClientMethodMapper.java

```diff
@@ -51,6 +51,11 @@ protected JavaVisibility methodVisibility(ClientMethodType methodType, boolean h
                     visibility = NOT_GENERATE;
                     break;
 
+                case PagingSync:
+                case PagingAsync:
+                    visibility = VISIBLE;
+                    break;
+
```

Comment:
I will probably remove it in another PR (after test). Your change of logic in ClientMethodMapper in this PR is correct, and fluent should not generate pageable API with optional but not with Context (it was a bug in the "sync-stack" PR, found yesterday but didn't have time to adjust).